src/components/Header.js

```diff
@@ -1,10 +1,57 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({ changeUser, newMessage }) {
+  const EUNKO = "https://img.techpowerup.org/200908/eun.png";
+  const COOL = "https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png";
+  const userCheck = () => {
+    if (newMessage.user) {
+      return <Image src={EUNKO} />;
+    } else {
+      return <Image src={COOL} />;
+    }
+  };
+
+  const displayName = () => {
+    if (newMessage.user) {
+      return "고은";
+    } else {
+      return "정쿨";
+    }
+  };
+
+  return (
+    <Wrapper>
+      <ChangeButton onClick={changeUser}>{userCheck()}</ChangeButton>
+      <ShowUser>
```

Comment:
`ShowUser`같은 동사형보다는 `UserInfo` 같은 네이밍이 더 적절하지 않을까요?ㅎㅅㅎ
```suggestion
      <UserInfo>
```

Reply:
아 이부분은 네이밍이 좋지 않았네요 수정할게요!